src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java

```diff
@@ -25,77 +21,14 @@ public DisplayScheduleCommand(String toDisplay) {
     public void execute(Ui ui, NoteList notes, TaskList tasks, FileStorage storage) {
         LocalDate currDate = LocalDate.now();
         if (isDisplayWeek) {
-            displayWeekSchedule(currDate, tasks);
+            scheduleUi.displayWeekSchedule(currDate, tasks);
         } else {
-            if (getTaskCountForToday(tasks, currDate) == 0) {
+            if (scheduleUi.getTaskCountForToday(tasks, currDate) == 0) {
                 System.out.println("You have nothing on for today!");
             } else {
                 System.out.println("Here is your schedule for today:");
-                displayDaySchedule(currDate, tasks);
+                scheduleUi.displayDaySchedule(currDate, tasks);
             }
```

Comment:
You could consider placing this section before the case for when there are no tasks in the day. Something to do with making the happy path prominent:
https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent